src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/Extensions/Name+Extension.swift

```diff
@@ -24,9 +24,9 @@ extension Name {
 	var standardizedName: String {
 		return [standardizedGivenName, standardizedFamilyName].formatted()
 	}
-
+	
 	var groupingStandardizedName: String {
-		return standardizedName
+		return trimmedStandardizedName
 			.replacingOccurrences(of: "\\s+", with: " ", options: .regularExpression)
 			.replacingOccurrences(of: "<+", with: "<", options: .regularExpression)
```

Comment:
Is this still needed? After trimming "<" this line should never be executed?

Reply:
Trimming is done from leading and trailing, this regular expression checks if we have more than one < and it replaces with a single <

Reply:
Eg: in case when standardizedGivenName is THOMAS<<ARMIN<

After trimming, we shall get THOMAS<<ARMIN
and with this regex, we will get THOMAS<ARMIN while grouping